src/wallet.test.ts

```diff
@@ -524,6 +568,35 @@ describe('wallet', () => {
       const promise = pify(engine.handle).call(engine, payload);
       await expect(promise).rejects.toThrow('Invalid input.');
     });
+
+    it('should not throw if request is permit with undefined value for verifyingContract address', async () => {
+      const { engine } = createTestSetup();
+      const getAccounts = async () => testAddresses.slice();
+      const witnessedMsgParams: TypedMessageParams[] = [];
+      const processTypedMessageV4 = async (msgParams: TypedMessageParams) => {
+        witnessedMsgParams.push(msgParams);
```

Comment:
Nit: This variable isn't used either
```suggestion
      const processTypedMessageV4 = async (msgParams: TypedMessageParams) => {
```